docker/Dockerfile-base

```diff
@@ -103,28 +115,7 @@ RUN /opt/conda/bin/pip install --no-cache-dir \
 # docker==5.0.3 \
 # adversarial-robustness-toolbox==1.9.1
 
-# TODO: pydub not installed. Do we need?
-#    pydub==0.24.1 \  
-
-WORKDIR /workspace
-
-########## PyTorch 1 Dev #################
-
-FROM armory-pytorch-base AS armory-pytorch-dev
-ARG armory_version
-
-COPY . /armory_dev/
-RUN /opt/conda/bin/pip install /armory_dev/ --no-cache-dir
-
-WORKDIR /workspace
-CMD tail -f /dev/null
-
-########## PyTorch 1 Release #################
-
-FROM armory-pytorch-base AS armory-pytorch
-ARG armory_version
-
-RUN /opt/conda/bin/pip install armory-testbed==${armory_version} --no-cache-dir
+# pydub not installed. Do we need?
+#    pydub==0.24.1 \
```

Comment:
We will likely need to create a "dataset-builder" docker image, and this will probably need to go there (as well as apache-beam).